doc/manual/manual.tex

```diff
@@ -7934,9 +7918,31 @@ \subsubsection{Inner core convection}
 Figure~\ref{fig:inner-core-regimes} shows examples of the three regimes with $Ra=3000, \mathcal{P}=1000$ (plume convection), 
 $Ra=10^5, \mathcal{P}=0.01$ (translation), $Ra=10, \mathcal{P}=30$ (no convection).
 
+\begin{figure}[h]
+    \begin{center}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e0P0modif.png}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e2P-1modif.png}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e5P4modif.png}
+   \vspace{1cm}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e0P0modif.png}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e2P-1rescalemodif.png}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e5P4rescalemodif.png}
+    \caption{Convection regimes in the inner core for different values of $Ra$ and $\mathcal{P}$. From left to right: no convection, translation, plume convection; the 2D slices of the top are with the default temperature scale while at the bottom an adaptative scale is used.}
+    \label{fig:inner-core-regimes}
+       \end{center}
+\end{figure}
+:w
+\vspace{0.3cm}
+\textbf{Mesh refinement.}
+The temperature is set at 1 et the boundary, and for the translation case, a large temperature gradient can be imposed at the boundary layer, especially for the translation regime. For this, a specific refinement can be set for defining correctly the boundary layer at the inner core boundary. 
```

Comment:
Maybe it would make more sense to formulate that in a different way:
The temperature is set to 0 at the outer boundary, and a large temperature gradient can develop in the boundary layer, especially for the translation regime. The adaptive mesh refinement allows it to resolve this layer at the inner core boundary.